.storybook/themes/news.scss

```diff
@@ -2156,23 +2156,14 @@
 						button-medium: (
 							padding: 0 var(--global-spacing-3),
 						),
+						icon: (
+							fill: var(--global-black),
```

Comment:
For these colors, should we use global-black and global-neutral-7, or color-primary and color-primary-hover? @nschubach @cedric066

Reply:
I'd double check with @pardistaherzadeh ... but Figma gives them unique colors so I don't think this is appropriate.  Figma doesn't appear to have hover states for these though:
![image](https://github.com/WPMedia/arc-themes-blocks/assets/2287238/84ed59ed-be26-4053-8a7c-e407ce9de31d)

Reply:
@blakeganderson I'm going to update to alias tokens now

@nschubach I'll send Pardis a DM and ask

Reply:
Pardis just confirmed to me that it should match the [review doc](https://paper.dropbox.com/doc/Themes-hover-states-audit--B400R8DxsjiVy0YImyMBZ0B4Ag-JyHvxHuZttlIfj5BzjTI3)